content/monitors/notifications.md

```diff
@@ -212,55 +213,59 @@ https://app.datadoghq.com/dash/integration/system_overview?tpl_var_scope=host:{{
-
-**Fillby, sizeby, filter**: Fill by can be defined by adding fillby:avg:<MetricName>.  Size by can be defined by adding sizeby:avg:<SecondMetricName>.  Filter can be used to specify a specific integration (i.e. Cassandra, mysql, apache, snmp, etc) by adding filter=<integration_name>
-{{< img src="monitors/notifications/hostmap_url.png" alt="hostmap_url" responsive="true" popup="true" >}}
+  ```
+  https://app.datadoghq.com/infrastructure/map?fillby=avg%3Acpuutilization&sizeby=avg%3Anometric&filter=cassandra 
+  ```
+  The above link has more customizable options than your standard System Dashboard.  Here you have additional variables to define.  Most common variables passed into this URL are the following **Fillby, sizeby, filter**: 
+
+  * `fillby` is defined by adding `fillby:avg:<MetricName>`.  
+  * `sizeby` is defined by adding `sizeby:avg:<SecondMetricName>`.
+  * `filter` is used to specify a specific integration (i.e. Cassandra, mysql, apache, snmp, etc) by adding `filter=<integration_name>`
+  {{< img src="monitors/notifications/hostmap_url.png" alt="hostmap_url" responsive="true" popup="true" style="width:70%;">}}
+  The above colors fill the hexagons by `system.cpu.system`, it sizes the hexagons by `system.cpu.stolen` and add a filter to only include Cassandra hosts.  
+  
 
-The above colors fill the hexagons by `system.cpu.system`, it sizes the hexagons by `system.cpu.stolen` and add a filter to only include Cassandra hosts. 
+*  **Integration Dashboards**- If you are building Application or Integration specific Monitors, link to that specific Integration Dashboard as well as adding a scope for the host that triggered the monitor.  
+  In the example below all that is necessary to populate is the `<integration_name>` section for something like Cassandra, apache, SNMP, etc as well as providing the scope for the offending host:
+  ```
+  https://app.datadoghq.com/dash/integration/<integration_name>?tpl_var_scope=host:{{host.name}}
+  ```
+  {{< img src="monitors/notifications/integration_url.png" alt="integration_url" responsive="true" popup="true" style="width:70%;">}}
 
-**Integration Dashboards**- If you are building Application or Integration specific Monitors, you can link to that specific Integration Dashboard as well as adding a scope for the host that triggered the monitor.  In the example below all that is necessary to populate is the *<integration_name> section for something like Cassandra, apache, SNMP, etc as well as providing the scope for the offending host:
+* **Manage Monitors Page** – To link to a Manage monitors page that displays all of the monitors for the host in question, define a link like below:  
+  ```
+  https://app.datadoghq.com/monitors/manage?q=scope:host:{{host.name}}
+  ```
+  The above links to all monitors for this host. You have other options available to further refine the link.  
```

Comment:
Should it read `to further refine the link` or `to further refine the query`?

Reply:
further refine the link, but I changed the sentence